CONTRIBUTORS

```diff
@@ -65,6 +64,25 @@ Andrew Auclair
 Jochen Sprickerhof
 Jesse Hoogervorst
 Aron Budea
-zhuzeitou
+Tou Zhu
 Daniel Grunwald
-
+Dan Rosser
+Eric Eichhorn
+Jeremy Rifkin
+Jouke Witteveen
+Marc Schlatter
+Marco Rebhan
+Mert Övün
+Mikhail Khachayants
+Nicolau Leal Werneck
+Sergei Trofimovich
+Seyyed Soroosh Hosseinalipour
+Felix Wang
+Thomas Beutlich
+Étienne Dupuis
+lanthora
+siren186
+Lara Dzivdzanovic
+Milan Tepic
+Peter Klotz
+nitram96 <wolhert@gmail.com>
```

Comment:
Let's not put emails here. Basically, I always try to find person's real name, if possible

Reply:
I also managed to find real names for many of the contributors. Not for this one, unfortunately.

Reply:
Removed e-mail.